src/components/rating/RatingContext.tsx

```diff
@@ -0,0 +1,19 @@
+import { ComponentProps, createContext, useContext } from 'react';
+
+export type Size = 'sm' | 'md' | 'lg';
+
+export type RatingContext = ComponentProps<'svg'> & {
```

Comment:
Why use the `ComponentProps<'svg'>` props? I don't see it used in the child components

Reply:
oh yeah I forgot to remove that.